server/cli/init/base-file-sources/hooks.js

```diff
@@ -1,41 +1,44 @@
 'use strict';
 
 /* eslint-disable no-var, prefer-arrow-callback */
-var path = require('path');
-var cucumber = require('cucumber');
 var constants = require('../../../constants')
-var outputDir = path.join(__dirname, '../', constants.REPORT_DIR)
-var fs = require('fs');
+var cucumber = require('cucumber');
+var cucumberHtmlReport = require('cucumber-html-report');
+var log = require('npmlog');
+var moment = require('moment');
+var path = require('path');
+var Promise = require('bluebird');
+
+var fs = Promise.promisifyAll(require('fs'));
  /* eslint-disable new-cap */
```

Comment:
I believe the lint override is needed as "By default both newIsCap and capIsNew options are set to true."